tools/gulptasks/dashboards/dist-examples.js

```diff
@@ -1,18 +1,36 @@
-/* eslint-disable no-console */
+/*
+ * Copyright (C) Highsoft AS
+ */
 
-const Gulp = require('gulp');
-const { join } = require('node:path');
-const { writeFile, readFile, rm, lstat } = require('node:fs/promises');
-const { getDemoBuildPath } = require('../dist-examples.js');
 
-const yargs = require('yargs');
+const fs = require('node:fs/promises');
+const gulp = require('gulp');
+const path = require('node:path');
 
-const TARGET_DIRECTORY = join('build', 'dist');
 
-async function readJSONFile(filePath) {
-    const fileContent = await readFile(filePath);
-    return JSON.parse(fileContent);
-}
+/* *
+ *
+ *  Constants
+ *
+ * */
+
+
+const HELPME = `
+Highcharts Dashboards - Dist Examples Task
+==========================================
+
+Requires demo.path to be set in git-ignore-me.properties. This should be set to the output folder of the highcharts-demo-manager frontend. Example: '../highcharts-demo-manager/frontend/tmp/';
+
+Resources:
+* https://github.com/highcharts/highcharts-demo-manager/tree/master/frontend
```

Comment:
https://github.com/highcharts/highcharts-demo-manager/tree/master/frontend URL does not exist